setup_priority-after-axp192 (#737)
esphome/esphome-devices: src/docs/devices/M5Stack-M5StickC-PLUS/index.md

    pdm: true
    id: mic
```

> ## Workaround for using devices powered with 5V  on the HY2.0-4P port

Reviewer: ```suggestion
## Workaround for using devices powered with 5V on the HY2.0-4P port
```

> The 5V power on the HY2.0-4P is fed by the axp192. Therfore these devices must be initialized some time after the axp192 has started.

Reviewer: ```suggestion
The 5V power on the HY2.0-4P is fed by the axp192. Therefore these devices must be initialized some time after the axp192 has started.
```

> ```yml

Reviewer: ```suggestion
```yaml
```

i2c:

>    - id: bus_gove

Reviewer: ```suggestion
   - id: bus_grove
```

     sda: GPIO32
     scl: GPIO33
     scan: True
     frequency: 100kHz
     ## Start after the axp192 has powered up

>      ## In case an additional delay is needed this component may be helpfull:

Reviewer: ```suggestion
     ## In case an additional delay is needed this component may be helpful:
```

     ## https://github.com/ssieb/esphome_components/tree/master/components/boot_delay
     setup_priority: 500

sensor:

>  ## Example: anything connected to "i2c_id: bus_gove" should have a lower setup priority then bus_gove.

Reviewer: ```suggestion
 ## Example: anything connected to "i2c_id: bus_grove" should have a lower setup priority than bus_grove.
```

  - platform: scd30
    setup_priority: 490

>     i2c_id: bus_gove

Reviewer: ```suggestion
    i2c_id: bus_grove
```
